crates/bins/src/bin/datadog-static-analyzer-git-hook.rs

```diff
@@ -49,6 +110,19 @@ fn main() -> Result<()> {
 
     opts.optopt("", "default-branch", "default branch", "main");
 
+    opts.optopt(
+        "",
+        "sha-start",
+        "sha start",
+        "077a3609dbff7c390efe54820e0bda3536685605",
```

Comment:
This will take whatever `git2` supports for taking a sha. It works for a long sha. And generally, users take a long sha since it it taken from a git hooks script. So i'd say it's not really relevant unless we have specific customer feedback on this.